babel.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {
+      targets: {
+        browsers: [
+          'IE 11',
+          'last 5 Chrome versions',
+          'last 5 Firefox versions',
+          'last 5 Safari versions',
+          'last 5 Edge versions',
+        ]
+      },
+      useBuiltIns: 'entry',
+    }],
+    '@babel/preset-react',
+  ],
+};
```

Comment:
Changed from `.babelrc` to `babel.config.js` (from directory babel config to project babel config). That way, consumers of this library will use their own babel config to process the files, instead of ours.